UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.API.FunctionalTests/appsettings.json

```
@@ -21,17 +21,28 @@
         "BaseUrl": "",
         "SubscriptionKey": ""
     },
+
     "POSWebjobApiConfiguration": {
-        "UserName": "",
-        "Password": "",
-        "BaseUrl": "",
-        "IsRunningOnLocalMachine": false
+        "UserName": "$pos-dev-lxs-webapp",
+        "Password": "H7DNr6Tfknvvp1vEkMlm4bY1AYeBEbpkbsP6wbSXFMCxQ2qhamXDJqExZxJn",
+        "BaseUrl": "https://pos-dev-lxs-webapp.scm.azurewebsites.net/api/triggeredwebjobs/POSFulfilmentWebJob/run",
```

This will be set trough terraform file transform no need to pass it
```suggestion
        "UserName": "",
        "Password": "",
        "BaseUrl": "",
```

---

updated . Thanks